src/main/webapp/app/entities/event-attendee/event-attendee-create.scss

```diff
@@ -0,0 +1,4 @@
+.transport-checkbox {
+  height: 25px !important;
+  margin-top: 0px;
+}
```

Comment:
maybe can put inside event-module.scss?

Reply:
Resolved in fe845d284e2cb16f075baa6dbb9d9c1eba3efa68